.env.example

```diff
@@ -9,6 +9,6 @@ SAML_CALLBACK_URL="http://italia-backend/assertionConsumerService"
 SAML_ISSUER="http://italia-backend"
 SAML_ATTRIBUTE_CONSUMING_SERVICE_INDEX=1
 PRE_SHARED_KEY="12345"
-ALLOW_NOTIFY_IP_SOURCE_RANGE="::ffff:172.19.3.0/112"
+ALLOW_NOTIFY_IP_SOURCE_RANGE="::ffff:ac13:1/112"
```

Comment:
I'll add documentation for every parameter here and/or in the README.
Why the ipv6 form ?